pkg/manager/member/pd_member_manager.go

```diff
@@ -619,6 +621,32 @@ func getNewPDSetForTidbCluster(tc *v1alpha1.TidbCluster, cm *corev1.ConfigMap) (
 		})
 	}
 
+	var additionalVolumeClaims []corev1.PersistentVolumeClaim
+	if len(tc.Spec.PD.StorageVolumes) > 0 {
```

Comment:
Could you please define a function for this code block and then we can call it for PD, TiDB, and TiKV.

Reply:
ok